.github/workflows/pythonapp.yml

```diff
@@ -28,21 +28,18 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install -r requirements.txt
+        pip install -r requirements/test.txt .
```

Comment:
does the "." add something here?

Reply:
I was surprised too. basically without "." it was not installing hummingbird, but only the requirements.